README.md

```diff
@@ -89,7 +89,9 @@ sudo xcode-select -s /Applications/Xcode_12.5.1.app/Contents/Developer
 
```

Comment:
```suggestion
The ```ulimit``` command fixes an issue related to shell resource usage. 
```

More exposition in docs for little things like this is helpful imo. especially when your system is set up a bit differently